crates/release_plz/src/commands.rs

```diff
@@ -0,0 +1,69 @@
+mod check_updates;
+mod completions;
+mod config;
+mod init_github;
+mod manifest;
+mod pull_request;
+mod release;
+mod repo;
+mod schema;
+mod set_version;
+mod update;
+
+use clap::ValueEnum;
+use serde::Serialize;
+
+use self::{
+    completions::Completions, init_github::InitGithub, pull_request::PullRequest, release::Release,
+    set_version::SetVersion, update::Update,
+};
+
+#[derive(clap::Subcommand, Debug)]
+pub enum Command {
+    #[command(alias = "init", long_about)]
+    InitGithub(InitGithub),
+    Update(Update),
+    #[command(name = "pr", aliases = ["pull-request", "release-pr"])]
+    PullRequest(PullRequest),
+    SetVersion(SetVersion),
+    Release(Release),
+    #[command(alias = "generate-completions")]
+    Completions(Completions),
```

Comment:
`completions` is a convention used in a number of commands for this functionality. alias for backwards compat